tools/c7n_azure/c7n_azure/resources/apimanagement.py

```diff
@@ -45,3 +48,57 @@ class resource_type(ArmResourceManager.resource_type):
             'resourceGroup'
         )
         resource_type = 'Microsoft.ApiManagement/service'
+
+
+@ApiManagement.action_registry.register('resize')
+class Resize(AzureBaseAction):
+    """
+    Action to scale api management resource.
+    Required arguments: capacity in units and tier (Developer, Basic, Standard or Premium).
+
+    :example:
+    This policy will resize api management to Premium tier with 8 units.
+
+    policies:
+      - name: resize-api
+        resource: azure.api-management
+        filters:
+          - type: value
+            key: name
+            value: test-api
+        actions:
+          - type: resize
+            tier: Premium
+            capacity: 8
+
+    """
+
+    schema = type_schema(
+        'resize',
+        required=['capacity', 'tier'],
+        **{
+            'capacity': {'type': 'number'},

```

Comment:
nit: you can add min\max value here (if there are any)

Reply:
I don't know if it's not going to change over time or depends (will depend) on the location. Thoughts?